src/components/VmsList/VmCardList.js

```diff
@@ -28,7 +28,7 @@ const VmCardList = ({ vms, alwaysShowPoolCard, fetchMoreVmsAndPools }) => {
 
   // Filter the Pools (only show a Pool card if the user can currently 'Take' a VM from it)
   const filteredPools = vms.get('pools')
-    .filter(pool => alwaysShowPoolCard || (pool.get('vmsCount') < pool.get('maxUserVms') && pool.get('size') > 0 && filterVms(pool, filters)))
+    .filter(pool => (alwaysShowPoolCard || (pool.get('vmsCount') < pool.get('maxUserVms') && pool.get('size') > 0)) && filterVms(pool, filters))
```

Comment:
```suggestion
    .filter(pool => 
      (alwaysShowPoolCard || (pool.get('vmsCount') < pool.get('maxUserVms') && pool.get('size') > 0)) 
      && filterVms(pool, filters)
    )
```

Good catch, nothing like parens causing problems!

I'm only suggesting an alternate layout so it's a bit easier to eye-ball the filter condition bits.  I'm ok with leaving it on one line, but my eye/brain/code-reading coordination prefers the multiline style. WDYT?